bridgetown-website/src/_components/shared/head.liquid

```diff
@@ -9,13 +9,12 @@
 <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
 <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
 
-<link rel="preload" href="{% webpack_path css %}" as="style" />
+<link rel="stylesheet" href="{% webpack_path css %}" />
```

Comment:
I moved these about a little :)

Requesting the preloading the stylesheet, then requesting it right after seemed to not make a big performance boost. I also moved the future page prefetches below the JS so that'll start downloading before we worry about the next pages.